api/src/__tests__/widgetMutations.test.ts

```diff
@@ -173,6 +176,57 @@ describe('messenger connect', () => {
     mock.restore();
   });
 
+  test('creates new visitor log', async () => {
+    const mock = sinon.stub(utils, 'sendRequest').callsFake(() => {
+      return Promise.resolve('success');
+    });
+
+    const logUtilsMock = sinon
+      .stub(logUtils, 'sendToVisitorLog')
+      .callsFake(() => {
+        return Promise.resolve('ok');
+      });
+
+    const response = await widgetMutations.widgetsMessengerConnect(
+      {},
+      {
+        brandCode: _brand.code || '',
+        visitorId: '123'
+      }
+    );
+
+    expect(response.customerId).toBeUndefined();
+    expect(response.visitorId).toBe('123');
+
+    logUtilsMock.restore();
+    mock.restore();
+  });
+
+  test('creates new customer when logger is not running', async () => {
```

Comment:
where is the new customer checking code?

Reply:
there is a piece of code in line 224. if the customer is not created, there is no customerId